runtests/settings.py

```diff
@@ -100,6 +99,7 @@
 
 ACTSTREAM_SETTINGS = {
     'MANAGER': 'testapp.streams.MyActionManager',
+    'VERB_TRANSFORMER':'actstream.tests.base.TestVerbTransformer',
```

Comment:
This whole feature needs to be reflected in the documentation, explaining why this may be useful and a usage example how to define my own `VerbTransformer`.

Reply:
Thanks for your kindly review! It's  very helpful.